db/schema.rb

```diff
@@ -15,13 +15,13 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "collections", id: :serial, force: :cascade do |t|
+  create_table "collections", force: :cascade do |t|
     t.string "concept_id", null: false
```

Comment:
We discussed migrating daac up to the collection/granule.  Why did you decide not to pursue that?

Reply:
I'll do this with regard to testing performance, want to get a base metric first.